### PR TITLE
added a note on how to safely reuse the `uv_write_t` object.

### PR DESCRIPTION
**Pull request by neevek**
Added a note in the doc to describe how a `uv_write_t` object can be safely reused.

### CODE REVIEW

File: docs/src/stream.rst [hunk 1]
```diff
@@ -26,7 +26,11 @@ Data types
 
 .. c:type:: uv_write_t
 
-    Write request type.
+    Write request type. Careful attention must be paid for resuing objects of
```
Comment (cjihrig) [inline docs/src/stream.rst]: for -> when
resuing -> reusing

File: docs/src/stream.rst [hunk 2]
```diff
@@ -26,7 +26,11 @@ Data types
 
 .. c:type:: uv_write_t
 
-    Write request type.
+    Write request type. Careful attention must be paid for resuing objects of 
+    this type. When a stream is in non-blocking mode, write requests sent 
+    with ``uv_write`` will be queued, resuing objects at this point is an
```
Comment (cjihrig) [inline docs/src/stream.rst]: Can you end the sentence after queued, and start a new sentence with Reusing (spelling again).

Comment (cjihrig) [inline docs/src/stream.rst]: I think you can drop "an"

**Comment by neevek** [comment]
Hi @cjihrig , I modified the commit as you suggested, and just made an --amend commit.

**Comment by cjihrig** [comment]
You would need to reopen this PR. Also, the diff currently says there are 336 changed files.

**Comment by neevek** [comment]
@cjihrig I opened a new PR here https://github.com/libuv/libuv/pull/927

**Pull request closed without merging**